### PR TITLE
Re-estimate project size

### PR DESCRIPTION
**Pull request by 2686747**
#974: estimated

### CODE REVIEW

**Comment by karato** [comment]
@2686747 Thanks, I will ask someone to do a code review here, thanks for your pull request

**Comment by karato** [comment]
@hdouss do this review

**Comment by HDouss** [comment]
@2686747 We should try to stick to "Don't Look Back" best practice. The scope you are suggesting is exactly the same as the one before. Please see: https://github.com/teamed/est

File: est/003.est [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  4: Google+ integration
+  5: User login
+  6: User logout
+  7: User account
+  8: Email notifications
+  9: Starting bouts
+  10: Inviting others to bouts
+  11: Adding attachments to bouts
+  12: Markdown support
+  13: Unread messages support
```
Comment (HDouss) [inline est/003.est]: @2686747 What does `Unread messages support` functionality mean ?

**Comment by HDouss** [comment]
@2686747 Please see 2 comments above.

File: est/003.est (hunk 1 above)
Comment (2686747) [inline est/003.est]: @HDouss Your message will be marked as "viewed" if user seen its.

Comment (HDouss) [inline est/003.est]: @2686747 I am not sure this is part of the scope, Or else, it meant something else.

**Comment by 2686747** [comment]
@HDouss Thanks a lot. Please, check 0c6c9f8

**Comment by HDouss** [comment]
@2686747 Thanks.

**Comment by HDouss** [comment]
@rultor merge.

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by karato** [comment]
@elenavolokhova please, review this issue for QA compliance, as per [par.24](http://at.teamed.io/policy.html#24)

**Comment by karato** [comment]
@rultor please deploy

**Comment by elenavolokhova** [comment]
@HDouss 
According to our [quality rules](http://www.teamed.io/qa.html):
 > The code reviewer found at least three problems in the code.
 
 There were only two issues found during review.
Please confirm that you will try to find at least three problems next time.

**Comment by HDouss** [comment]
@elenavolokhova I confirm.

**Comment by elenavolokhova** [comment]
@karato Quality is acceptable here.

**Comment by karato** [comment]
> @karato Quality is acceptable here.

@elenavolokhova thanks for the QA review, we'll work better next time

**Comment by karato** [comment]
@hdouss 10 mins was added to the account of @elenavolokhova (for QA review), in transaction 78555398... **15 mins** sent to your balance (ID `78555423`), many thanks! It took 46 hours and 29 mins.... added +15 to your rating, now it is equal to [+2848](http://www.netbout.com/b/36032?open=rating)

**Pull request merged**